expirationd.lua

```diff
@@ -298,27 +303,28 @@ end
 -- Expiration daemon management functions
 -- ========================================================================= --
 
--- Run a named task
+-- Run a scheduled task to check and process (expire) tuples in a given space.
+--
 -- params:
 --   name             -- task name
 --   space_id         -- space to look in for expired tuples
 --   is_tuple_expired -- a function, must accept tuple and return
 --                       true/false (is tuple expired or not),
 --                       receives (args, tuple) as arguments
 --   options = {      -- (table with named options)
---     * process_expired_tuple -- applied to expired tuples, receives
---                                (space_id, args, tuple) as arguments
---     * on_full_scan_start    -- call function on starting full scan iteration
---     * on_full_scan_complete -- call function on complete full scan iteration
---     * on_full_scan_success  -- call function on success full scan iteration
---     * on_full_scan_error    -- call function on error full scan iteration
---     * args                  -- passed to is_tuple_expired and
---                                process_expired_tuple() as additional context
---     * tuples_per_iteration  -- number of tuples will be checked by one iteration
---     * full_scan_time        -- time required for full index scan (in seconds)
---     * iteration_delay       -- max sleep time between iterations (in seconds)
---     * full_scan_delay       -- sleep time between full scans (in seconds)
---     * force                 -- run task even on replica
+--     * process_expired_tuple -- Applied to expired tuples, receives (space_id, args, tuple) as arguments;
+--                                can be nil: by default, tuples are removed.
+--     * on_full_scan_start    -- Function to call before starting a full scan iteration.
+--     * on_full_scan_complete -- Function to call after completing a full scan iteration.
+--     * on_full_scan_success  -- Function to call after successfully completing a full scan iteration.
+--     * on_full_scan_error    -- Function to call after terminating a full scan due to an error.
+--     * args                  -- Passed to is_tuple_expired and
+--                                process_expired_tuple() as additional context.
+--     * tuples_per_iteration  -- Number of tuples to check in one batch (iteration); default is 1024.
+--     * full_scan_time        -- Time required for a full index scan (in seconds).
+--     * iteration_delay       -- Max sleep time between iterations (in seconds).
+--     * full_scan_delay       -- Sleep time between full scans (in seconds).
+--     * force                 -- Run task even on replica.
```

Comment:
How change with formatting is related to this commit?

Reply:
commit msg: `Comments, readme, and responses to errors are presented in a more uniform form.`